apollos-church-api/src/server.js

```diff
@@ -90,6 +90,22 @@ app.get('/version', cors(), (req, res) => {
   }
 });
 
+app.use('/', (req, res, next) => {
```

Comment:
do you need the `/` - does this work without providing a path? I'm trying to remember how the matching works.... both requests to `/` and `/graphql` are valid and I'm not sure if the mobile client uses one or the other

Reply:
Ah ok, I don't think I need that path.  Didn't want to over-specify!

Reply:
Cool - just tested it works at both root and `/graphql`